datasync/chaindatafetcher/kafka/config.go

```diff
@@ -40,6 +40,7 @@ const (
 	DefaultTopicResourceName    = "en-0"
 	DefaultMaxMessageBytes      = 1000000
 	DefaultRequiredAcks         = 1
+	DefaultSegmentSize          = 1000000 // 1 MB
```

Comment:
1MB = 1024 * 1024 Bytes, isn't it?

Reply:
Right, but the default value of kafka configuration is 1000000.

Reply:
This value may be modified. The smaller segment size would be better for the performance.

Reply:
MiB is 1024^2. MB is 1000^2.